frontend/src/admin/components/clone-user-access/clone-user-access.scss

```diff
@@ -0,0 +1,6 @@
+@import "../../../styles/main";
+
+.c-clone-user-access {
+  max-width: 200px;
+  margin: 0 0 24px;
+}
```

Comment:
This styling does not seem to be related to the `CloneUserAcces`, but rather the layout that it resides in.